interface/patient_file/summary/demographics.php

```diff
@@ -1746,7 +1747,7 @@ function setMyPatient() {
   }
   $picture_url = "";
   //begin file uploading
-  $destination_directory = "../../../profile_pictures/";
+  $destination_directory = "../../../sites/".$_SESSION['site_id']."/profile_pictures/";
```

Comment:
$GLOBALS['OE_SITES_BASE'] again, but frankly, some setting of vars for these directories could be done in one place.